tests/test_cli.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from typer.testing import CliRunner
+
+from minari.cli import app
+from minari.storage.local import delete_dataset, list_local_datasets
+from tests.dataset.test_dataset_download import get_latest_compatible_dataset_id
+
+
+runner = CliRunner()
+
+
+def test_list_app():
+    result = runner.invoke(app, ["list", "local", "--all"])
+    assert result.exit_code == 0
+    # some of the other columns may be cut off by Rich
```

Comment:
what is "Rich" as alluded to in this comment is that rich text?

Reply:
https://github.com/Farama-Foundation/Minari/blob/da8578c8df55beb5f00f477f1f2d15d9741fbbe7/minari/cli.py#L8

my initial plan was just check if the whole header is what is expected in the table but the table format changes depending on terminal size.  i'm pretty sure there is a way it could be done by specifying something related in rich but i think it might require changing more aspects of the cli.py file

Reply:
I think it's fine as-is, I didn't realize rich as actually a package we used in the CLI